balboa-http/src/main/scala/com/socrata/balboa/server/MainServlet.scala

```diff
@@ -37,9 +43,32 @@ class MainServlet extends ScalatraServlet
     }
   }
 
-  get("/entities*") {
-    val response = EntitiesRest(params)
-    contentType = response.contentType
-    response.result
+  get("/entities*")(getEntities)
+  def getEntities: ActionResult = {
+    contentType = json
+
+    val predicate: (String => Boolean) =
+      params.get("filter").map(Pattern.compile)
+        .map { pattern => (str: String) => pattern.matcher(str).matches }.getOrElse(_ => true)
+
+    timer("entities queries")({
```

Comment:
What about `entities-get` as the name? The truth is I'm not entirely sure everywhere this ends up downstream, but this name should (if I understand things correctly) be the name of the metric. I think it might be nice if it was key like.